src/core/waitForCoreSync.js

```diff
@@ -5,16 +5,24 @@ const wait = require('../util/wait');
  *
  * @typedef {waitForCoreSync}
  * @param {CoreService} coreService
+ * @param {updateSyncStatus} updateSyncStatus
```

Comment:
Please update doc  block

Reply:
OK, updated. Is JSDoc part of TypeScript? Or what is the purpose of this section?